packages/assets-controllers/src/TokensController.ts

```diff
@@ -667,144 +614,50 @@ export class TokensController extends BaseController<
     asset: Token,
     type: string,
     interactingAddress?: string,
-  ): Promise<AssetSuggestionResult> {
```

Comment:
I have checked and the contents of `AssetSuggestionResult.suggestedAssetMeta` are not used neither in mobile nor in extension.

As a matter of fact not even the address (`AssetSuggestionResult.result`) is being used, just awaited, so for all we cared we could even have `Promise<void>`.